packages/bun-internal-test/src/runner.node.mjs

```diff
@@ -558,7 +558,7 @@ function emojiTag(tag) {
     emojiText += "";
   }
 
-  if (tag.includes("x86") || tag.includes("_64") || tag.includes("amd64")) {
+  if (tag.includes("x86") || tag.includes("x64") || tag.includes("_64") || tag.includes("amd64")) {
```

Comment:
why has _64 been a part of this check? that is a very dubious way to check for x86_64.